library/kotlin/io/envoyproxy/envoymobile/Engine.kt

```diff
@@ -20,4 +20,11 @@ interface Engine {
    * Terminates the running engine.
    */
   fun terminate()
+
+  /**
+   * Flush the stats sinks outside of a flushing interval.
+   * Note: stat flushing is done asynchronously, this function will never block.
+   * This is a noop if called before the underlying EnvoyEngine has started.
+   */
+  fun flushStats()
```

Comment:
Just realized this, it'll make more sense to fold this method to `PulseClient`, no changes needed as for now, can be planned as a clean-up/follow-up change.